grails-app/controllers/org/broadinstitute/orsp/api/NewConsentGroupController.groovy

```diff
@@ -168,6 +171,24 @@ class NewConsentGroupController extends AuthenticatedController {
         collectionLinks
     }
 
+    def getProjectConsentGroups() {
+        UtilityClass.registerIssueForSampleDataCohortsMarshaller()
+        try {
+            LinkedHashMap consentGroups = consentService.findProjectConsentGroups(params.projectKey)
+            response.status = 200
+            JSON.use(UtilityClass.ISSUE_COMPLETE) {
+                render( consentGroups as JSON)
+            }
+        } catch(IllegalArgumentException e) {
+            response.status = 400
+            render([message: 'Error while trying to get project\'s Consent Groups. ' + e as JSON])
+        } catch(Exception e) {
+            log.error("Error while trying to get Consent Groups for ProjectKey: ${params.projectKey}. ", e.getMessage())
+            response.status = 500
+            render([message: 'Error while trying to get project\'s Consent Groups.' as JSON])
```

Comment:
nit here and above, you can avoid the escape, `\`, if you double quote the string.